bin/tessel-2.js

```diff
@@ -171,6 +171,11 @@ makeCommand('wifi')
   })
   .help('Configure the wireless connection');
 
+makeCommand('memory')
```

Comment:
Do we need to make this accessible to the user? It should hopefully rarely come up so I'd shy away from adding *more* commands.